geonode_mapstore_client/static/mapstore/configs/localConfig.json

```
@@ -2330,6 +2330,150 @@
                 "cfg": {
                     "wrap": true
                 }
+            },
+            {
+                "name": "LayerDetailViewer",
```

```suggestion
                "mandatory": true,
                "name": "LayerDetailViewer",
```

In order to make the plugin available by default add `mandatory: true`, or add it to pluginsConfig under `TOC` as `children`, so the user can select the plugin when adding `viewer` to map.

Currently, the plugin is not available on map with viewer existing already.